test/Website/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM microsoft/dotnet:1.0-runtime
+
+COPY bin/Release/netcoreapp1.0/publish /app
+WORKDIR /app
+ 
+EXPOSE 80/tcp
```

Comment:
I don't usually see protocol (tcp) is specified when a port is exposed, but this shouldn't matter